.github/workflows/pytest.yml

```
@@ -77,6 +77,11 @@ jobs:
           pytest -n 2 --dist loadfile --nbval --current-env --cov-append tutorials/
           echo "Exited with '$?'"
 
+      - name: test with WeldxFile
+        if: (matrix.os == 'ubuntu-latest') && (matrix.py == '3.8')
+        run: |
+          pytest ./weldx/tests/asdf_tests -n2 --dist loadfile --weldx-file-rw-buffer --weldx-file-rw-buffer-disp-header
```

```suggestion
          pytest -n2 --dist loadfile --weldx-file-rw-buffer --weldx-file-rw-buffer-disp-header --cov-append ./weldx/tests/asdf_tests
```
probably append coverage here ...